package.json

```diff
@@ -19,7 +19,7 @@
     "foundation-sites": "6.4.3",
     "history": "4.7.2",
     "lodash-es": "4.17.4",
-    "query-string": "6.0.0",
+    "query-string": "5.0.0",
```

Comment:
I have tried a new library to do the same in Half-Earth and seems to work better: https://github.com/ljharb/qs as it also supports nested properties parsing. So I think worth it to update it here.